setup.py

```diff
@@ -68,7 +68,7 @@ def get_requirements(requirements_file="requirements.txt"):
     ],
     packages=find_packages(exclude=["anitya.tests", "anitya.tests.*"]),
     include_package_data=True,
-    scripts=["files/anitya_cron.py", "anitya/sar.py"],
+    scripts=["anitya/check_service.py", "anitya/sar.py"],
```

Comment:
This service will run in a new pod in Openshift ?

Reply:
Yes, instead of Cronjob I will have new pod running.

Reply:
Ok cool :+1: